src/main/java/uk/gov/hmcts/reform/jobscheduler/model/errors/FieldError.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.jobscheduler.model.errors;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class FieldError {
+
+    @JsonProperty("field_name")
+    public final String fieldName;
+
+    public final String value;
```

Comment:
Should this field be named `errorMessage`?

Needs `@JsonProperty` annotation.

Reply:
renamed to `message`